com.vinszent.GnomeTwitch.json

```diff
@@ -0,0 +1,175 @@
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "configure",
+                    "commands": [
+                        "./configure.py --bootstrap"
+                    ]
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "Makefile",
+                    "commands": [
+                        "all:",
+                        "install:",
+                        "\tinstall -D ninja /app/bin/ninja"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "meson",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mesonbuild/meson.git",
+                    "branch": "0.33.0"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "configure",
+                    "commands": [ "" ]
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "Makefile",
+                    "commands": [
+                        "all:",
+                        "install:",
+                        "\tpython3 ./setup.py install --prefix=/app/"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "yasm",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
+                    "sha256": "3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f"
+                }
+            ]
+        }, {
+            "name": "gst-libav",
+            "cleanup": [
+                "/bin/*",
+                "/lib/peas-demo"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://anongit.freedesktop.org/gstreamer/gst-libav",
+                    "branch": "1.8"
+                }
+            ]
+        }, {
+            "name": "libpeas",
+            "cleanup": [
+                "/bin/*",
+                "/lib/peas-demo"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.gnome.org/libpeas",
+                    "branch": "libpeas-1.18.0"
+                }
+            ]
+        }, {
+            "name": "gnome-twitch",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mattiasb/gnome-twitch.git",
+                    "branch": "feature/flatpak"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "configure",
+                    "commands": [
```

Comment:
Why not add those as files on the local filesystem instead? I'd create a separate flatpak directory, and put all the files in there.